src/SessionSetup/ProcessLauncherWidget.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "SessionSetup/ProcessLauncherWidget.h"
+
+#include <QFileDialog>
+#include <QLineEdit>
+
+#include "OrbitBase/Logging.h"
+#include "ui_ProcessLauncherWidget.h"
+
+namespace orbit_session_setup {
+
+ProcessLauncherWidget::ProcessLauncherWidget(QWidget* parent)
+    : QWidget(parent), ui_(std::make_unique<Ui::ProcessLauncherWidget>()) {
+  ui_->setupUi(this);
+  ui_->ProcessComboBox->lineEdit()->setPlaceholderText("Process");
+  ui_->WorkingDirComboBox->lineEdit()->setPlaceholderText("Working directory");
+  ui_->ArgumentsComboBox->lineEdit()->setPlaceholderText("Arguments");
+  ui_->ErrorLabel->setText("");
+  ui_->gridLayout_2->setColumnStretch(0, 90);
```

Comment:
Is there a reason to not do this in the .ui file?

Reply:
Setting the placeholder text doesn't work on the combo boxes. As for the ErrorLabel, well if we don't display anything it's invisible in the Qt Creator, so I prefer to leave the default text as "ErrorLabel" and set it to "" here. I also had issues with setting the column stretch. Quite frankly, the less I use Qt Creator the better.